README.md

```diff
@@ -48,9 +56,6 @@ Base URL: http://stats.filspark.com/
 
   http://stats.filspark.com/participant//0x000000000000000000000000000000000000dEaD/reward-transfers
 
-- `GET /miners/retrieval-success-rate/summary?from=<day>&to=<day>`
-
-  http://stats.filspark.com/miners/retrieval-success-rate/summary
```

Comment:
why was this removed?

Reply:
Hmm, this looks like a mistake; thanks for flagging it!

Reply:
Found it. This was a duplicate entry, the endpoint is already documented on lines 19-21 above.

<img width="634" alt="Screenshot 2024-09-02 at 12 12 33" src="https://github.com/user-attachments/assets/270d8acb-cf5f-474f-ab80-7361841801bf">